StringCalculator/StringCalculator.cs

```diff
@@ -0,0 +1,89 @@
+using System;
+using System.Collections.Generic;
+
+namespace StringCalculator
+{
+    public class StringCalculator
+    {
+        public StringCalculator()
+        {
+
+        }
+
+        public int Add(string stringInput)
```

Comment:
Add in this case can be a static method and you don't need to initialise the StringCalculator

Reply:
yeah. static may be a good candidate here. We also need to be mindful of testing static methods. Many testing libraries do not support that. Does xunit support @mwGHburger

Reply:
yeah, static may be a good candidate here. If we do that, this class can become `final` as it will be utility class and we don't want anyone to extend it